src/flamegraph/color/palettes.rs

```diff
@@ -59,6 +59,31 @@ pub(super) mod perl {
     }
 }
 
+pub(super) mod python {
+    use crate::flamegraph::color::BasicPalette;
+
+    fn split_any_path(path: &str) -> impl Iterator<Item = &str> {
+        path.split(|c| c == '/' || c == '\\')
+    }
+
+    pub(in super::super) fn resolve(name: &str) -> BasicPalette {
+      if split_any_path(name).any(|part| part == "site-packages") {
+            return BasicPalette::Mem;
+        } else if split_any_path(name).any(|part| {
+            part.strip_prefix("python")
+                .or_else(|| part.strip_prefix("Python"))
+                .map_or(false, |version| {
+                    version.chars().all(|c| c.is_digit(10) || c == '.')
+                })
+        }) || name.starts_with("<")
+        {
+            // stdlib
+            return BasicPalette::Gray
```

Comment:
Why gray for the stdlib?

Reply:
It was looking a bit garish/fruit salad as it was and I wanted to draw attention away from stdlib calls, as you aren't going to be able to optimise calls into the standard library.

Reply:
Sort of — you might still be able to optimize by choosing different calls for example. Gray tends to mean "ignore this", which I don't think applies to "this code spent much of its time in this place in the standard library".

Reply:
I've switched this to Yellow, which is a python-y colour and doesn't stand out too much